.github/workflows/test.yaml

```diff
@@ -12,6 +12,8 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
       - run: npm ci
       - run: npm run lint
+      - run: npm run type
+      - run: git diff --exit-code
```

Comment:
Rather than add a “build” step. We now have a CI step that will fail if a diff is generated from running `npm run type`. This is closer to the flow I _think_ we want. I.e., rather than need to run a build to use the types… assert that the types were properly generated before allowing the merge.